SignalMessaging/utils/OWSSwiftUtils.swift

```diff
@@ -7,15 +7,15 @@ import Foundation
 /**
  * We synchronize access to state in this class using this queue.
  */
-func assertOnQueue(_ queue: DispatchQueue) {
+public func assertOnQueue(_ queue: DispatchQueue) {
     if #available(iOS 10.0, *) {
         dispatchPrecondition(condition: .onQueue(queue))
     } else {
         // Skipping check on <iOS10, since syntax is different and it's just a development convenience.
     }
 }
 
-func owsFail(_ message: String) {
+public func owsFail(_ message: String) {
```

Comment:
These methods need to be public so that we can use them in the main app and the SAE.